gitgud/levels/rampup/_relrefs2/instructions.txt

```diff
@@ -0,0 +1,9 @@
+Now that you are a master with the ^ operator, let's learn how to use the ~ operator.
+>>>
+The tilde operator is used much the same way as the ^ operator; although it can be more concise than the ^ operator. The usage of the ~ operator is this: "git checkout HEAD~<num>".
+>>>
+Using this command will place the HEAD pointer at the commit node that is <num> commits in the past. So, if I want to check out the node 5 commits before HEAD, I would use: "git checkout HEAD~5". This command also works with branch names instead of HEAD.
```

Comment:
Just like ^, this command also works with other types of references like branch names, commit hashes and tags, instead of just HEAD